subsys/bootloader/bl_crypto/bl_crypto_cc310_common.c

```diff
@@ -14,34 +15,6 @@
 #define NRF_CRYPTOCELL NRF_CRYPTOCELL_S
 #endif
 
-/* These are needed by the cc310_bl code, but not
- * (always) included in the bootloader.
- */
-void *__weak memset(void *buf, int c, size_t n)
-{
-	for (size_t i = 0; i < n; i++) {
-		((u8_t *)buf)[i] = c;
-	}
-	return buf;
-}
-
-void *__weak memcpy(void *restrict d, const void *restrict s, size_t n)
```

Comment:
not needed anymore? Is this because we use stdlib from zephyr?

Reply:
I think it was moved to a different file. Think it was mentioned in one of the commit messages

Reply:
Ah yes found them in `bootloader.c`